circle.yml

```diff
@@ -153,23 +153,23 @@ jobs:
     docker:
       - image: ethereum/cpp-build-env:12-gcc-8
     environment:
-      CMAKE_OPTIONS: -DTOOLCHAIN=cxx17-pic
+      CMAKE_OPTIONS: -DCMAKE_CXX_STANDARD=17
```

Comment:
Is this PIC by default?

Reply:
Yes. All the EVMC libs are built as PIC. I made this change some time ago. None of this code is performance critical and this is convenient if someone what to include them in a shared library.

Reply:
I mean the cable toolchain was explicitly PIC, but I wasn't sure cmake sets PIC on by default.

Reply:
CMake does not set PIC by default.
- It will set PIC when building a shared library, but only for files directly included in the library (linked in static library will not become PIC magically).
- We are enabling PIC in EVMC static libs: https://github.com/ethereum/evmc/blob/master/lib/instructions/CMakeLists.txt#L15.
- You can always enable that with `-DCMAKE_POSITION_INDEPENDENT_CODE=TRUE`.